vision/vae_mnist/Project.toml

```
@@ -7,12 +7,12 @@ ImageIO = "82e4d734-157c-48bb-816b-45c225c6df19"
 ImageMagick = "6218d12a-5da1-5696-b52f-db25d2ecc6d1"
 Images = "916415d5-f1e6-5110-898d-aaa5f9f070e0"
 MLDatasets = "eb30cadb-4394-5ae3-aed4-317e484a6458"
-Parameters = "d96e819e-fc66-5662-9728-84c9c7592b0a"
 Plots = "91a5bcdd-55d7-5caf-9e0b-520d859cae80"
 ProgressMeter = "92933f4c-e287-5a05-a399-4b506db050ca"
 TensorBoardLogger = "899adc3e-224a-11e9-021f-63837185c80f"
 
 [compat]
-CUDA = "2.4"
-Flux = "0.11.4, 0.12"
-julia = "1.5"
+CUDA = "2.4, 3.12"
```

```suggestion
CUDA = "3.12"
```
What is 3.12 needed for? Just the latest version?

---

Just the latest. Flux current requires 3.